### DISCUSSION TITLE
List of Supported Hardware?

### DISCUSSION THREAD

**Discussion by kcobley** [ImpulseAdventure/GUIslice#375] [Q&A]
Hi there,

Is there a list of supported hardware? I looked for one on the wiki, but only found a vague " its works on all arduino variants" which is a statement that can get out of date.

Reason for asking is that the examples wont compile on my MKR1000, giving the following error:

fatal error: pgmspace.h: No such file or directory
     #include <pgmspace.h>

looks like a hardware compatibility problem?

 I edited the config file, selecting:
#include "../configs/ard-adagfx-ili9341-xpt2046.h"

as the closest match (i.e. SAMD 32 and matching my display and touch controller)

**Comment by Pconti31** [comment]
@kcobley As you should be aware the SAM MCUs do not have progmem support.  Try setting
```
  #define GSLC_USE_PROGMEM      0
```
inside your ../configs/ard-adagfx-ili9341-xpt2046.h file.
Paul--

**Reply by kcobley** [reply]
Thanks for the fast help. Looks like I have a different error now, so it may have sorted it.

I didn't know, and I hope not to have to know - in the spirit of Arduino. I wouldn't even know what progmem support is. For me that's the same as my board is not supported and that's reflected in the 'detect platform' code, which is not complete if it needs to do what you've asked me to add.

Don't get me wrong - I'm grateful for your support - you may have me running again once I fix the next error which looks more trivial (multiple Adafruit_gfx libraries). 

Thank-you.

**Reply by kcobley** [reply]
Just to confirm that it compiles without error after this change (and having removed the duplicate library).

Thanks again.
